app/views/report_mailer/tables_report.html.erb

```diff
@@ -0,0 +1,85 @@
+<%#
+- @tables: an array of tables, expects first "row" to be an options hash
+%>
+
+<style type="text/css">
+
+.table-number {
+  font-variant-numeric: tabular-nums;
+  text-align: right;
+}
+
+<%#
```

Comment:
Could we use design system styling?

https://designsystem.digital.gov/components/table/

We could make a separate SCSS bundle for the reports mailers and import just the one component?

Something like....

```scss
// app/assets/stylesheets/report-email.css.scss
@forward 'uswds-core';
@forward 'usa-table';
```

Reply:
good idea, added in 9d050cf07ec51c0e04ca1c86b8fa1220be23b73b

Reply:
Updated screenshot:

| screenshot |
| --- |
| <img width="605" alt="Screenshot 2023-09-18 at 9 41 59 AM" src="https://github.com/18F/identity-idp/assets/458784/42e2cbfd-28ec-4c64-8351-4a99c5a72b76"> |